frontend/components/profile/ViewProfile.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { View, Text, Button, Image, StyleSheet, Dimensions } from 'react-native';
+
+export default ViewProfile = ({ navigation }) => {
+  const dWidth = Dimensions.get('window').width;
+  const dHeight = Dimensions.get('window').height;
+
+  return (
+    <View style={styles.moreInfo}>
+      <Button title="Log Out" style={{
```

Comment:
react native <Buttons> are not flexible in styling, it won't reflect what's specified in style. Try substituting <Button> with <TouchableOpacity>. Any text on the button wrap it in a <Text> and put inside <TouchableOpacity>